tests/integration/fixtures/non-updatable-views-filtered.graphql

```diff
@@ -0,0 +1,17 @@
+query {
+  noInsertMutation: __type(name: "InsertNonMutationViewPayload") {
```

Comment:
Clever :blush:

In the future I’m probably going to use [this](http://graphql.org/docs/api-reference-utilities/#printschema) function so that we can better test the contents of our generated GraphQL schema.